### PR TITLE
Reduce number of IDE errors and wrong imports

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

**Comment by bdattoma** [comment]
I personally don't like having every time the Library name in the keyword:
- it reduces readability of the code
- it makes writing code slower

Of course, if the majority of the team agrees with this choice, let's merge it

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,4 @@ results/
 __pycache__/
 robocop.log
 infrastructure_configuration.yaml
-ods-ci.log*
```
[the remark this answers is not in the document]
Comment (kornys) [reply .gitignore:21]: because it is a duplicate, it is already defined on the top in .gitignore

**Comment by Frawless** [comment]
@bdattoma I am not sure if it reduces readability of the code especially if there are two sources from where the method could be called. Both Pycharm and VScode raised it as a warning which made us thinking why and what exactly is going on within the step. Unless there is a better solution (some config for robot lib?) I think this is a good solution

**Comment by bdattoma** [comment]
>why and what exactly is going on within the step


I think there is nothing wrong going on with the test step itself. We set the library search order (i.e., `Set Library Search Order    SeleniumLibrary`) in the suite setup so that Robot Framework can run the code. IDE are not able to capture the search order set in the test. I'm not aware of IDE configs about it

**Comment by kornys** [comment]
I also didn't find any config for robotframework-lsp which can respect library order

**Comment by kornys** [comment]
@bdattoma I just found setting in robotframework-lsp to disable this lint warning -> So I will remove SeleniumLibrary prefix from methods and leave just import fixes. And update doc what it shouldbe setup in robotgramework-lsp

**Comment by kornys** [comment]
@bdattoma @jgarciao jenkins was triggered with this change and no issue.

**Review by bdattoma — APPROVED** [review]
lgmt, thanks! which setting of robotframework-lsp are you referring to?

**Comment by kornys** [comment]
> lgmt, thanks!
> 
> > just found setting in robotframework-lsp to disable this lint warning
> 
> which setting of robotframework-lsp are you referring to?

"robot.lint.keywordResolvesToMultipleKeywords": false

**Review by tarukumar — APPROVED** [review]

**Pull request merged**